src/test/java/team7/hrbank/unit/department/util/DepartmentRepositoryUtil.java

```diff
@@ -0,0 +1,97 @@
+package team7.hrbank.unit.department.util;
+
+import net.datafaker.Faker;
+import org.springframework.util.StringUtils;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class DepartmentRepositoryUtil {
```

Comment:
테스트 픽스처이니 클래스명은 `DepartmentFixture`라는 이름이 좀더 적절할 것 같아요.  
```suggestion
public class. DepartmentFixture {
```